src/audiomixerboard.cpp

```diff
@@ -619,7 +620,7 @@ void CChannelFader::UpdateSoloState ( const bool bNewOtherSoloState )
 
 void CChannelFader::SetChannelLevel ( const uint16_t iLevel )
 {
-    plbrChannelLevel->SetValue ( iLevel );
+    plbrChannelLevel->SetValue ( iLevel * dPreviousFaderLevel * 0.01 );
```

Comment:
Hrm, I don't understand why this is necessary and whether this is safe to do as this touches a generic function unconditionally of the usage of the new feature?

Reply:
It appears to be part of an independent feature, and was merged from another of @menzels branches called "change level indicater to post-fader". Should probably be in a separate PR.